scripts/uploadBundles.ts

```diff
@@ -122,7 +124,7 @@ async function updateManifest(newVersions: Map<string, string>) {
         [dirName]: {
           ...initial.packages[dirName],
           default: value,
-          versions: [...(initial.packages[dirName]?.versions || []), value],
+          versions: [...(initial.packages[dirName]?.versions || []), {version: value, timestamp}],
```

Comment:
Is the server ready to handle both the new and old format?

Reply:
yes, we back ported the timestamps to the manifest and updated the server